lib/index.mli

```diff
@@ -56,7 +56,7 @@ val get_build_history_with_time :
   owner:string ->
   name:string ->
   gref:string ->
-  (string * string * float option) list
+  (string * string * string * float option) list
```

Comment:
Maybe we need to introduce a type to describe the `refInfo` as we get more and more string. Something like:
```ocaml
type ref_info = {
  title: string;
  hash: string;
  gref: string;
  started_at: float option
}
```